### PR TITLE
Usetags

### PR DESCRIPTION
**Pull request by lewismarshall**
Need to test...

### CODE REVIEW

File: pkg/etcd/etcd.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ type ClientConfig struct {
 
 var (
 	Timeout = 5 * time.Second
-	MaxTransactionTime = 20 * time.Second
+	MaxTransactionTime = 120 * time.Second
```
Comment (marcinc) [inline pkg/etcd/etcd.go]: Do these vars ever change? If not make them const.

File: pkg/kmm/cloudprovider.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	}
+	var supported = false
+	node, supported = cloud.Node()
+	if supported {
+		log.Printf("Cloud Provider Initialized [%q]", cloud.ProviderName())
+	} else {
+		return nil, fmt.Errorf("Cloud Provider set [%q] but node interface not supported", cloud.ProviderName())
+	}
+	return node, nil
+}
```
Comment (marcinc) [inline pkg/kmm/cloudprovider.go]: NL :)

File: pkg/kmm/cmd/saveassets.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	)
+	if err != nil {
+		log.Fatal(err)
+		os.Exit(-1)
+	}
+}
+
+func init() {
+	RootCmd.AddCommand(saveassetsCmd)
+}
```
Comment (marcinc) [inline pkg/kmm/cmd/saveassets.go]: NL.

File: pkg/kmm/cloudprovider.go (hunk 2 above)
Comment (lewismarshall) [inline pkg/kmm/cloudprovider.go]: :)

**Comment by lewismarshall** [comment]
I'll set up a linter and do the changes as a seperate PR...

**Pull request merged**